app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java

```diff
@@ -118,9 +114,6 @@ public void onClick(View v) {
                     public void fileSelected(final File file) {
 // Here, thisActivity is the current activity
```

Comment:
This comment is not well-aligned with the text

Reply:
Alligned